sdk/nodejs/ec2/instanceType.ts

```diff
@@ -21,341 +21,341 @@
 
 /* tslint:disable:max-line-length ordered-imports variable-name */
 
+import { InstanceType } from ".";
+
 export module InstanceTypes {
```

Comment:
Add top-level deprecation message?

```suggestion
/** @deprecated Use InstanceType instead. */
export module InstanceTypes {
```